cl/recap/tasks.py

```diff
@@ -1720,11 +1729,32 @@ def fetch_docket(self, fq_pk):
         newly_enqueued = enqueue_docket_alert(d_pk)
         if newly_enqueued:
             send_alert_and_webhook(d_pk, start_time)
-    msg = "Successfully got and merged docket. Adding to Solr as final step."
-    mark_fq_status(fq, msg, PROCESSING_STATUS.SUCCESSFUL)
     return result
 
 
+def recap_fetch_webhook_check(fq: PacerFetchQueue) -> None:
```

Comment:
Couldn't this just be put into `send_recap_fetch_webhook_event`, and then we rename that one to be `send_recap_fetch_webhooks`, so it's similar to the name of `send_docket_alert_webhooks`?